site/ar_LB.all.json

```diff
@@ -1602,5 +1602,29 @@
   },
   "wcag_aria_label_social_letterboxd": {
     "other": "قم بزيارتنا على Letterboxd"
+  },
+  "validate_email_about": {
+    "other": "لإنشاء حساب ، يجب علينا أولاً التحقق من عنوان بريدك الإلكتروني. أدخل بريدك الإلكتروني وسنرسل دعوة تسجيل حساب."
+  },
+  "validate_email_success": {
+    "other": "تم إرسال بريد إلكتروني للتحقق إلى {{.email}} . يرجى التحقق من بريدك الإلكتروني واتباع تعليمات إنشاء الحساب."
```

Comment:
These variables should be PascalCase e.g. `{{.Email}}`.

Reply:
OK I can do that. Out of interest, is this a consistency thing (which is all good) or will they not work using `camelCase`? I only used `camelCase` since that meant we can use niceties in js like object shorthand.

E.g.

```js
i18n('validate_email_success', { email });
```

vs having to specify the keys:

```js
i18n('validate_email_success', { Email: email });
```

Since is js `camelCase` is the standard.

Reply:
Memory is foggy but I think it's related to Golang struct public fields for use with go-i18n.  On the Relish side it's probably just for consistency though.